fpga_interchange/chip_info.py

```diff
@@ -739,6 +742,8 @@ def append_bba(self, bba, label_prefix):
             bba.ref(self.field_label(label_prefix, field))
             bba.u32(len(getattr(self, field)))
 
+        bba.u32(1 if self.out_of_site_clusters else 0)
```

Comment:
this looks like a breaking bba change, I think, it should also bump https://github.com/SymbiFlow/python-fpga-interchange/blob/baa113424c1803704e3123d2c7a298a728e98e3b/fpga_interchange/chip_info.py#L1139

Reply:
Indeed, thanks

Reply:
Done